idl_parser/idl_lexer.py

```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/env python
+# Copyright (c) 2013 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
```

Comment:
We should include Chromium's license file in the root of this repo. (I'm assuming that webcore and idl_parser share the same Chromium license).

Reply:
They're ever so slightly different so I just included a LICENSE both in webcore and in idl_parser.

https://chromium.googlesource.com/chromium/src/+/refs/heads/main/LICENSE (for idl_parser)
https://chromium.googlesource.com/chromium/blink/+/refs/heads/main/LICENSE (for webcore)